src/components/EmojiPicker/EmojiPickerMenu/index.js

```diff
@@ -8,7 +8,7 @@ import CONST from '../../../CONST';
 import ONYXKEYS from '../../../ONYXKEYS';
 import styles from '../../../styles/styles';
 import * as StyleUtils from '../../../styles/StyleUtils';
-import emojis from '../../../../assets/emojis';
+import emojiAssets from '../../../../assets/emojis';
```

Comment:
I had to rename a few variables like these

**Reason**
In the previous class component
there was this `emojis` default import + there was a `this.emojis` (ref)

But in functional component there will be conflict becaouse
`this.emojis` is converted to `emojis` (created using useRef)